ComAmazonawsDynamodb/codegen-patches/dotnet/dafny-4.1.0.patch

```diff
@@ -0,0 +1,19 @@
+diff --git b/ComAmazonawsDynamodb/runtimes/net/Generated/TypeConversion.cs a/ComAmazonawsDynamodb/runtimes/net/Generated/TypeConversion.cs
+index 72738b20..83ba3510 100644
+--- b/ComAmazonawsDynamodb/runtimes/net/Generated/TypeConversion.cs
++++ a/ComAmazonawsDynamodb/runtimes/net/Generated/TypeConversion.cs
+@@ -10809,7 +10809,13 @@ namespace Com.Amazonaws.Dynamodb
+         }
+         public static Wrappers_Compile._IOption<Dafny.ISequence<Dafny.ISequence<char>>> ToDafny_N3_com__N9_amazonaws__N8_dynamodb__S10_Projection__M16_NonKeyAttributes(System.Collections.Generic.List<string> value)
+         {
+-            return value == null ? Wrappers_Compile.Option<Dafny.ISequence<Dafny.ISequence<char>>>.create_None() : Wrappers_Compile.Option<Dafny.ISequence<Dafny.ISequence<char>>>.create_Some(ToDafny_N3_com__N9_amazonaws__N8_dynamodb__S23_NonKeyAttributeNameList((System.Collections.Generic.List<string>)value));
++            // BEGIN MANUAL EDIT
```

Comment:
Confused why this is showing up, I thought the version of smithy-dafny locked by the submodule already includes this change?

Reply:
I was surprised as well.
I ran this command from the root of the repo:
```
make polymorph_code_gen DAFNY_VERSION=4.1.0 POLYMORPH_OPTIONS=--update-patch-files
```
And this was the result.

Reply:
Running `make polymorph_code_gen DAFNY_VERSION=4.1.0 POLYMORPH_OPTIONS=--update-patch-files` in `ComAmazonawsDynamodb` did not change anything.

Reply:
Okay, since the actual code isn't changing I'm comfortable figuring this out later